### DISCUSSION TITLE
GitHub Models Playground - Code Blocks Invisible in Dark Mode

### DISCUSSION THREAD

**Discussion by rusherman** [community/community#141753] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have been using the GitHub Models Playground and have noticed an issue when using the dark mode theme. The code blocks generated in the playground are extremely difficult to see. This makes it challenging to work with and understand the generated code.
I believe it would be extremely beneficial if the visibility of code blocks could be improved in the dark mode theme. This would enhance the user experience and make it easier for developers to utilize the powerful features of the GitHub Models Playground.

**Comment by MorelliniMatteo** [comment]
It seems that the issue you're experiencing with the GitHub Models Playground in dark mode is related to low contrast in the code blocks, which makes them hard to read. Improving the visibility of the code blocks in dark mode is a valid suggestion, as it would definitely improve the overall user experience.

Here’s what you can do in the meantime:

Adjust Browser Settings: Some browsers or browser extensions allow you to modify the appearance of websites, including increasing the contrast or adjusting colors. You might want to explore browser extensions like "Dark Reader," which allows you to fine-tune dark mode settings.

Report the Issue to GitHub: You can raise this issue by submitting feedback directly to GitHub. This ensures that the GitHub team is aware of the problem and can implement improvements to the dark mode UI. GitHub is generally responsive to community feedback, and improvements to UI/UX are often prioritized.

Switching to Light Mode Temporarily: As a temporary solution, you can switch to light mode when working with code blocks in the GitHub Models Playground until the issue is resolved.

If you feel that this impacts your work significantly, submitting this as feedback is the best course of action, as GitHub may address it in future updates.
